src/lib.rs

```diff
@@ -3,6 +3,5 @@ pub mod db;
 pub mod error;
 pub mod lightning;
 pub mod nip33;
-pub mod nip59;
```

Comment:
I'm looking this fix https://github.com/rust-nostr/nostr/commit/70db575d51965240aab1e1b3f1edab782c0ef625

Changing the pubkey of the rumor is not the only reason I created my own nip59 library, I also did it because I needed to decrypt the content with a conversation key, which I couldn't do it with sdk, so first time I built my own decrypt with ck function and I tested with data encrypted with the sdk and it didn't work.

Please remove all code related with nip59 from this PR called `rate_user` and let's figure it out how to do nip59 improvement in another PR